pkg/util/orchestrator/pod.go

```diff
@@ -67,6 +68,9 @@ func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterID
 		// additional tags
 		podModel.Tags = append(tags, fmt.Sprintf("pod_status:%s", strings.ToLower(podModel.Status)))
 
+		// collect priorityClassName
+		podModel.Tags = append(tags, fmt.Sprintf("priority_class:%s", strings.ToLower(podModel.PriorityClass)))
```

Comment:
This information will already be there given by this PR https://github.com/DataDog/datadog-agent/pull/9172 in the tagger, therefore you don't need to add this tag here. I suggest to do this in our backend for `datadog-cluster-agent` payloads

Reply:
+ The name doesn't seem to match i.e. from above PR: `kube_priority_class`